src/Service_Command.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+use EE\Utils;
+
+/**
+ * Manages global containers of EasyEngine.
+ *
+ * ## EXAMPLES
+ *
+ *     # Restarts global nginx proxy containers
+ *     $ ee service restart nginx-proxy
+ *
+ * @package ee-cli
+ */
+class Service_Command extends EE_Command {
+
+	/**
+	 * @var array Array of containers defined in global docker-compose.yml
+	 */
+	private $whitelisted_containers = [
```

Comment:
Although we are interacting with containers, we are not directly interacting with the containers themselves.
As such you've not used the container name but the service name mentioned in docker-compose.yml.

So although there is no functional change, but I think it would be better if we renamed almost all instances of the word *container* with *service*.